sdks/go/pkg/beam/runners/universal/universal.go

```diff
@@ -78,6 +78,20 @@ func Execute(ctx context.Context, p *beam.Pipeline) error {
 	return err
 }
 
+const (
+	urnLegacyProgressReporting = "beam:protocol:progress_reporting:v0"
+	urnMultiCore               = "beam:protocol:multi_core_bundle_processing:v1"
+)
+
+func goCapabilities() []string {
+	capabilities := []string{
+		urnLegacyProgressReporting,
+		urnLegacyProgressReporting,
+	}
+	capabilities = append(capabilities, graphx.KnownStandardCoders()...)
+	return capabilities
+}
+
```

Comment:
You can return straight from the append and save a line.

Reply:
Done. I wasn't sure what the convention was--the semantics of append (may modify the first argument or return a new value) seemed odd to me.

Reply:
Fair enough.
Technically append always returns a new "slice" but the backing array for the data may be reused. It's possible to have aliasing problems if done poorly, but that's not a risk here.
 It mostly comes down to the difference between the used length, and the capacity of the slice. If there's enough capacity, it will simply mutate the backing slice.

To avoid the aliasing, as a rule, one should avoid using an slice one didn't allocate as the base for append.